components/identities/availableIdentities.tsx

```diff
@@ -99,13 +97,13 @@ const AvailableIdentities = ({ tokenId }: { tokenId: string }) => {
     fetchProfilePic();
   }, [identity]);
 
-  const hideActionsHandler = (state: boolean) => {
-    if (state == true) {
+  useEffect(() => {
```

Comment:
You can simplify this logic by just doing `setHideActions(!isIdentityADomain)` or `setShowActions(isIdentityADomain)`